star-navigation/src/components/simulation/simulation-view.tsx

```diff
@@ -7,6 +7,7 @@ import BackIcon from "../../assets/back-icon.svg";
 import { clsx } from "clsx";
 
 import css from "./simulation-view.scss";
+import { Landscape } from "./landscape";
```

Comment:
You broke your import after the css import rule :)

Reply:
Thanks 🥹 I'm gonna fix it in the next PR. :)